### PR TITLE
kvo version bump

### PR DESCRIPTION
**Pull request by thorfour**
Bumping kvo version 

Before creating your PR, please make sure to add the appropriate GitHub labels
like e.g. `run-smoke-tests` + `platform/<xxx>`. For more details see
[tests/README.md](../tests/README.md).

(In case you don't have permissions to add labels, please ask a
[Maintainer](../MAINTAINERS).)

### CODE REVIEW

File: config.tf [hunk 1]
```diff
@@ -88,7 +88,7 @@ variable "tectonic_container_images" {
     kubednsmasq                  = "gcr.io/google_containers/k8s-dns-dnsmasq-nanny-amd64:1.14.5"
     kubedns_sidecar              = "gcr.io/google_containers/k8s-dns-sidecar-amd64:1.14.5"
     kube_version                 = "quay.io/coreos/kube-version:0.1.0"
-    kube_version_operator        = "quay.io/coreos/kube-version-operator:v1.8.4-kvo.7"
+    kube_version_operator        = "quay.io/coreos/kube-version-operator:v1.8.7-kvo.1"
```
Comment (diegs) [inline config.tf]: we also need to update the kubernetes version below.

(and someone should also update the tectonic version, cc @robszumski )

Comment (thorfour) [reply config.tf]: bumped kubernetes version

**Comment by diegs** [comment]
test upgrade from 1.8.4-tectonic.3 1.8.4-tectonic.3

**Comment by diegs** [comment]
@yifan-gu not sure if upgrade tests also require the tectonic version to be bumped? if so we can probably just bump it to 1.8.7-tectonic.1 (if that's our expectation cc @robszumski )

**Pull request merged**